masterlist.yaml

```diff
@@ -28,10 +28,84 @@ globals:
         text: '[Postagem mais recente do LOOT](%1%).'
     subs: [ 'https://loot.github.io/latest-thread/' ]
 
-groups: []
+groups:
+- name: &dlcGroup DLC
+
+- name: &unofficialPatches
```

Comment:
The name has an anchor, but no value. Adding a value should fix the CI validation.